### PR TITLE
added RAM_DISK_EXCLUDE to prevent copy of gzips back to RAM disk

### PR DESCRIPTION
**Pull request by PoohCook**

### CODE REVIEW

File: log2ram [hunk 1]
```diff
@@ -47,10 +47,14 @@ syncFromDisk () {
         exit 1
     fi
 
-    if [ -x "$(command -v rsync)" ]; then
-        rsync -aXv --inplace --no-whole-file --delete-after $HDD_LOG/ $RAM_LOG/ 2>&1 | tee -a $LOG2RAM_LOG
+    if [ ! -z "$RAM_DISK_EXCLUDE" ]; then
+        find $HDD_LOG/ -type f ! -name $RAM_DISK_EXCLUDE -exec cp -rfup {} $RAM_LOG/ \; 2>&1 | tee -a $LOG2RAM_LOG
```
Comment (jaketenten) [inline log2ram:51]: there isn't any rsync option for this?

**Review by jaketenten — COMMENTED** [review]
just a couple of questions

File: log2ram [hunk 2]
```diff
@@ -47,10 +47,14 @@ syncFromDisk () {
...
     else
-        cp -rfup $HDD_LOG/ -T $RAM_LOG/ 2>&1 | tee -a $LOG2RAM_LOG
+        if [ -x "$(command -v rsync)" ]; then
```
[the remark this answers is not in the document]
Comment (PoohCook) [reply log2ram:53]: no...  this is bash

File: log2ram (hunk 1 above)
Comment (PoohCook) [reply log2ram:51]: No...  we don't even have rsync on our system...

Comment (PoohCook) [reply log2ram:51]: With the hack to get rid of logrotate.timer, I've dropped the idea of pushing these changes back to the master for log2ram...

File: log2ram (hunk 2 above)
[the remark this answers is not in the document]
Comment (PoohCook) [reply log2ram:53]: So...  this is an sh script not bash...   I'm unclear if elif is supported in sh...  either way, I tested this, I'd rather not have to to setup the test again just for syntactic elegance...

**Review by jaketenten — APPROVED** [review]

**Pull request merged**